force-app/main/default/triggers/LeadTrigger.trigger

```diff
@@ -0,0 +1,11 @@
+Trigger LeadTrigger on Lead(after insert, after update){
+
+    if(trigger.isAfter){
+        if(trigger.isInsert){
+            leadTriggerHandler.createLeadTask(trigger.New, trigger.OldMap);
```

Comment:
Just a note, in the _insert_ context, you won't have an oldMap to work with. This will pass in as a null parameter. Not a big deal in this case since you don't use it for an insert.